src/encoded/tests/data/master-inserts/static_section.json

```diff
@@ -958,7 +958,16 @@
     },
 
 
-
+    {
+        "name" : "basic-carousel.slide",
+        "uuid" : "442c8aa0-dc6c-43d7-814a-854af4600028",
+        "body" : "<BasicCarousel\r\n    slides={[\r\n\t\t\t{\r\n                'img': '\/static\/img\/Metadata_structure_slides\/Slide01.png',\r\n                'alt': 'Slide 1 alt text'\r\n            },\r\n            {\r\n                'img': '\/static\/img\/Metadata_structure_slides\/Slide02.png',\r\n                'alt': 'Slide 2 alt text'\r\n            },\r\n            {\r\n                'img': '\/static\/img\/Metadata_structure_slides\/Slide03.png',\r\n                'alt': 'Slide 3 alt text'\r\n            }\r\n\t]}\r\n    slideWidth={720}\r\n    slideHeight={540}\r\n    showSlideCount={true}\r\n    autoPlay={true}\r\n    key='random-basic-carousel-key'\r\n\/>",
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary escape character: \/. (no-useless-escape)](https://app.codacy.com/app/4dn/fourfront/pullRequest?prid=4495549)

Reply:
@alexkb009 your comment is very detailed and informative, thank you for your time and sharing it. I learned the SVG styling by the way. (The bottom dots were ok on local machine by old styling, idk why looking green square and black dot overlapped.) all fixes you point out is ok but just a short comment for carousel props: we didn’t want to expose nuka-carousel’s own properties to user explicitly. If we move to a new carousel package in the future, an abstraction prevents the need to patch the already inserted data.

Reply:
Ah makes sense - good idea!
Will try to revert a little bit and maybe leave in slideHeight/slideWidth/autoplay as regular props. For simplicity/less-changes, probably will keep `props.carouselOptions` but just heavily discourage its usage.

Reply:
Hm for some reason am now getting `span.paging-dot` instead of `svg.paging-dot` lol. Prev styling def makes sense more re: span.paging-dot. Trying to figure out what causes Nuka carousel to render 1 vs other..

Reply:
Just added styling for the span.paging-dot also. Whether span or svg is used might have had to do with the cellAlign: center or slideHeight/slideWidth being sent to carousel as props. No idea. Which is strange and doesn't seem like those things should be related. 

Either way, we have CSS that should handle either case now hopefully.